pkg/karmadactl/cmdinit/utils/util.go

```diff
@@ -126,6 +126,18 @@ func ioCopyN(outFile *os.File, tr *tar.Reader) error {
 	return nil
 }
 
+// PathExists check if the file path exists
+func PathExists(path string) error {
+	_, err := os.Stat(path)
+	if err == nil {
+		return nil
+	}
+	if os.IsNotExist(err) {
+		return err
+	}
+	return nil
```

Comment:
Hello, I wanna why if `err != nil && !os.IsNotExist(err)`, we not return the err but return nil?

how about like this:

```
func PathExist(path string) error {
    _, err := os.Stat(path)
    if err != nil && !os.IsExist(err) {
        return err
    }
    return nil
}
```